src/nvim/ops.c

```diff
@@ -2361,11 +2324,29 @@ static void free_yank_all(void)
  *
  * Return FAIL for failure, OK otherwise.
  */
-int op_yank(oparg_T *oap, int deleting, int mess)
+int op_yank(oparg_T *oap, bool mess) {
```

Comment:
Open brace should be on the next line. `FUNC_ATTR_NONNUL_ALL`. Also, looks like it can return a bool.